src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  display: flex;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  width: 200px;
+  max-height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product {
+    margin: 0 auto 4px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-accent-color;
+    line-height: 18px;
+  }
+
+  &__code {
+    line-height: 14px;
+    margin: 0 0 16px;
+    font-size: 10px;
+    color: $secondary-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+    width: 100%;
+    height: 40px;
+    color: #fff;
+    background-color: $blue-accent-color;
+    border: 1px solid $blue-accent-color;
+    border-radius: 5px;
+    transition: 0.5s;
```

Comment:
move transition tin into variable and list propetries that you are transitioning (color and background-color)